### ISSUE TITLE
[BUG] Dark Cloud 2 - Saving to a folder-type memory card while running through Steam causes slowdown.

### ISSUE THREAD

**Issue by MageKing17** [PCSX2/pcsx2#4076]
**Describe the bug**
When PCSX2 is run through Steam and the saving progress bar appears in Dark Cloud 2 while saving to a folder-type memory card, regardless of whether you're saving a regular savegame or a photo album, the emulator slows down drastically (to around 30% or lower, at least on my system), without an increase in EE/GS/VU percentage (in fact, EE and GS percentage go down, presumably because it's doing the same amount of effort per frame at a lower framerate; VU percentage is 0% before and after). The slowdown only occurs with saving progress bars, not loading progress bars.

**To Reproduce**
Run PCSX2 through Steam. Boot Dark Cloud 2 and save the game or a photo album to a folder-type memory card. Observe the sudden slowdown. The moment it goes away, the speed will return to normal.

**Expected behavior**
The game shouldn't slow down while the saving progress bar is on screen.

**GS Settings**
Happens regardless of hardware or software modes, even on default settings.

**Emulation Settings**
Happens regardless of speedhacks, including MTVU and Instant VU1. Even happens on "safest" preset.

**System Info (please complete the following information):**
 - PCSX2 Revision: `v1.7.0-dev-780-gade01ecb8` through `v1.7.0-dev-822-gaf8987975` (current latest build)
 - OS: Windows 10
 - CPU: AMD Ryzen 7 1700X (8 cores @ 3.4 GHz)
 - GPU: GeForce GTX 970

Note that the problem *doesn't* occur in `v1.7.0-dev-771-g44b700a6f`, the first build available on the buildbot before `dev-780`.

**Comment by tadanokojin** [comment]
Couldn't replicate on latest dev as of today (861).
So I went ahead and tried on a build you identified (781) and I can't replicate it there either.

Launched the game via steam with a folder memory card and saved in the first accessible save point (repair shop in prologue)
The result was a consistent 100% speed throughout the loading bar's presence.

**Comment by MrCK1** [comment]
Closing since we are unable to reproduce this issue

**Comment by MageKing17** [comment]
So I finally got around to testing this again; first, I confirmed that the problem does not occur in the latest build (`v1.7.2869`). Second, I confirmed that I could still make the problem occur in `dev-822`, which I could. Then I realized a necessary reproduction step I missed in the original post: the whole reason I was running through Steam was to make my DualShock 4 pretend to be an XInput device for force feedback purposes (rather than jump through the hoop of using DS4Windows or similar). Sure enough, I unplugged the DS4 (and set up some keyboard controls in PCSX2) and `dev-822` no longer exhibited the issue. Just for the sake of my own edification, I then checked `v1.7.2731` (the last version I used before the Qt interface started being offered in nightly builds) and the problem did not occur there, either.

I was now too curious for my own good, though, so I started doing a binary search through nightly builds. `dev-1664`, problem occurs. `v1.7.2198`, problem occurs... or does it? Checking closer, `v1.7.2198` has something neither `dev-1664` nor `v1.7.2731` has: both 32-bit and 64-bit download options. And the 64-bit version *doesn't* exhibit the problem. So, it looks like this was ultimately fixed by switching to 64-bit. For further confirmation, I tried the 32-bit version of `v1.7.2484` (the last version before 32-bit builds were removed) and, indeed, the problem still occurs there.

So, given that 32-bit builds aren't available anymore (and, AIUI, the plan is to never offer them again) this can safely be considered fixed.
